modules/ROOT/pages/exchange-api.adoc

```diff
@@ -259,6 +260,50 @@ https://anypoint.mulesoft.com/exchange/api/v2/assets/:groupId/:assetId/:version/
   }'
 ----
 
+[[update-mutable-data]]
+=== Update Mutable Data
+
+This example shows how to update the mutable data of an existing asset and describe the asset in the same request. The mutable data of an asset includes categories, custom fields, tags, and documentation pages.
```

Comment:
Doesn't the text in the second sentence belong in the associated concept topic, about what mutable data is and why you might want or need to update it?

Reply:
Rewrote to ensure that the text describes the example, and provides clear info about user goals.